### ISSUE TITLE
UI in MAC

### ISSUE THREAD

**Issue by eorione** [ahmadudin/electron-instaPy-GUI#81]
Hi!
I just download the .dmg but when I try to select the PATH it won't open and let me choose the instapy folder.
Additionally, I can close the UI, looks buggy. It is happening to me on 2 different macs.

The program wont shut down.

**Comment by ahmadudin** [comment]
Yeah, the app is deprecated and so outdated, just wait for an update.

**Comment by lorenzoPrimi** [comment]
@ahmadudin will this update arrive anytime? Do you need help on something?
